playwright.config.js

```diff
@@ -4,7 +4,10 @@ import { defineConfig } from '@playwright/test';
 export default defineConfig({
   expect: {
     toHaveScreenshot: {
-      threshold: 0, // 0.2 by default.
+      // 0.2, the default, produces too many false negatives. 0, on the other
+      // hand, produces too many false positives. The idea is for this number
+      // to be as close to 0 as possible without any false positives.
+      threshold: 0.05,
```

Comment:
@clintcs brought this over to your PR to minimize the changes.  I would have expected the `variant="secondary"` + `:hover` test to fail due to the change from a solid color to transparent, but looks like it didn't. We can chat about it on the side to get this dialed in more?

Reply:
I just pushed a [commit](https://github.com/CrowdStrike/glide-core/pull/831/commits/f61ab654be0e7518782d7b555b7e93df59d5a642) to #831 testing `0.03`. It [didn't](https://glide-core.crowdstrike-ux.workers.dev/add-form-controls-layout-right-split/visual-test-report) produce a false positive, which means we can go even lower. You might try `0.03` here to see if it's low enough to pick up your background change.

Reply:
That seems to have done the trick. Thank you!

Reply:
Yeah for sure. I'm good continuing to play with this until we dial it in. Thanks a bunch!